{{ cookiecutter.formal_name }}/briefcase.toml

```diff
@@ -15,4 +15,19 @@ icon.square.96 = "app/src/main/res/mipmap-xhdpi/ic_launcher.png"
 icon.square.144 = "app/src/main/res/mipmap-xxhdpi/ic_launcher.png"
 icon.square.192 = "app/src/main/res/mipmap-xxxhdpi/ic_launcher.png"
 
-# splash.750x1334 = "app/src/main/res/.../launch-750x1334.png"
+splash.160dp_mdpi = "app/src/main/res/drawable/launch.png"
+splash.240dp_mdpi = "app/src/main/res/drawable-sw480dp-mdpi/launch.png"
+splash.300dp_mdpi = "app/src/main/res/drawable-sw600dp-mdpi/launch.png"
+splash.360dp_mdpi = "app/src/main/res/drawable-sw720dp-mdpi/launch.png"
+splash.160dp_hdpi = "app/src/main/res/drawable-hdpi/launch.png"
+splash.240dp_hdpi = "app/src/main/res/drawable-sw480dp-hdpi/launch.png"
+splash.300dp_hdpi = "app/src/main/res/drawable-sw600dp-hdpi/launch.png"
+splash.360dp_hdpi = "app/src/main/res/drawable-sw720dp-hdpi/launch.png"
+splash.160dp_xhdpi = "app/src/main/res/drawable-xhdpi/launch.png"
+splash.240dp_xhdpi = "app/src/main/res/drawable-sw480dp-xdpi/launch.png"
+splash.300dp_xhdpi = "app/src/main/res/drawable-sw600dp-xdpi/launch.png"
+splash.360dp_xhdpi = "app/src/main/res/drawable-sw720dp-xdpi/launch.png"
+splash.160dp_xxxhdpi = "app/src/main/res/drawable-xxxhdpi/launch.png"
+splash.240dp_xxxhdpi = "app/src/main/res/drawable-sw480dp-xxxdpi/launch.png"
+splash.300dp_xxxhdpi = "app/src/main/res/drawable-sw600dp-xxxdpi/launch.png"
+splash.360dp_xxxhdpi = "app/src/main/res/drawable-sw720dp-xxxdpi/launch.png"
```

Comment:
typo - these three lines are missing the 'h' in 'xxxhdpi'

Reply:
Thanks for fixing these too!